src/Microsoft.Identity.Client/AppConfig/MatsConfig.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Identity.Client.AppConfig
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public class MatsConfig : IMatsConfig
```

Comment:
Is it necessary to have an interface over a simple data object?

Reply:
The nice thing about the interface is that it provides an immutable interface on top of the type, so we know that it can't be modified once sent into the system (see your request below to make it immutable).  so it enforces the contract to our internals that this type isn't modifiable.

---
In reply to: [270951391](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/955#discussion_r270951391) [](ancestors = 270951391)